### ISSUE TITLE
Add Handler to specific slide

### ISSUE THREAD

**Issue by kwelch** [FormidableLabs/spectacle#375]
This would allow for the ability to do a 'choose your own presentation' model. 

Created from conversations on [this thread](https://twitter.com/ken_wheeler/status/921038046853763077)

**Comment by kwelch** [comment]
I reviewed the PR and I see how you would be able to say that Slide 2 goes to Slide 5 with the `goTo` prop.

How would one setup a button on a slide that would allow you to go to either Slide 5 or Slide 3?

I apologize because I didn't describe this functionality well. 

Here is my attempt of a graph of what I am hoping to be able to do. 

```
      1
      |
      2
     /    \
    3     4
   / |      | \
  5 6    5 7
```

**Comment by carlospaelinck** [comment]
Yeah, we’re going to add support for a button that does effectively calls goTo and supports the slide number or identifier.

**Comment by kwelch** [comment]
Awesome! Thanks.

**Comment by carlospaelinck** [comment]
@kwelch Hopefully `<GoToAction />` gives you what you need. Two configs supported, standard button or custom rendered component.

**Comment by kwelch** [comment]
That is epic! Nice works. Thank you so much for this. Cannot wait to use it.
